packages/0x.js/src/0x.ts

```diff
@@ -71,11 +71,12 @@ export class ZeroEx {
      * @return  Whether the signature is valid for the supplied signerAddress and data.
      */
     public static isValidSignature(data: string, signature: ECSignature, signerAddress: string): boolean {
+        const normalizedSignerAddress = signerAddress.toLowerCase();
```

Comment:
@dekz I think we want to do these checks **after** we've asserted that it's an ETHAddressHex. Otherwise this could throw an unhelpful error if signerAddress is not a string :(

Reply:
hmm let me think about this because its the `isETHAddressHex` which is asserting it is a lowercase address, so passing into `isETHAddressHex` first would raise the assertion error.